build-system/server/app-utils.js

```diff
@@ -80,64 +82,81 @@ const isRtvMode = (serveMode) => {
   return /^\d{15}$/.test(serveMode);
 };
 
+const tsModule = Object.create(null);
```

Comment:
Curious: Why the global (vs. returning a new object from the function)?

Reply:
It's a cache.